### ISSUE TITLE
Can't type letters in remapped sequence

### ISSUE THREAD

**Issue by Wehrdo** [onivim/oni2#595]
Commonly, two-letter sequences like `jj` are remapped to the escape key to exit insert mode without stretching for the escape key. This currently does not work.

Steps to reproduce:
1. Run command `:inoremap jj <Esc>`
2. Enter insert mode via `i`
3. Type single `j`.

**Behavior**: `j` never appears
**Expected behavior**: `j` appears, unless immediately followed by a second `j` in a short time.

Basically, the remapped sequence should be time-limited, and cancelled if other letters are typed. Currently, if you type, for example, `j abcd j`, the `jj` sequence will be triggered, although it shouldn't.

**Comment by CrossR** [comment]
(Came across this issue whilst double checking the format to do this sort of stuff)

I believe this is fixed via https://github.com/onivim/oni2/pull/1497, though I haven't checked the Vim binding version, so could be worth leaving for that, or having a more specific issue later do deal with that.

**Comment by Wehrdo** [comment]
Haven't used Onivim in a while, but just tested out the latest version, and this is fixed! Thanks!

**Comment by Wehrdo** [comment]
Sorry, my mistake. See my edit above. Reopened.
